static/styles/widgets.css

```diff
@@ -167,3 +175,20 @@ button {
 .current-user-vote {
     background-color: hsla(156, 10%, 90%, 0.9);
 }
+
+.add-task-wrapper {
+    display: inline;
+    position: relative;
+    z-index: 1;
+
+    &:hover {
+        cursor: not-allowed;
```

Comment:
Hmm, why is the `cursor` declaration here?  It seems like we wouldn't not this cursor in the case it isn't disabled.

Reply:
When testing and using the todo, it was evident that the styling of the enabled button trumped the styling of the div so it functioned [normally](https://user-images.githubusercontent.com/36671046/143785490-87a6dbeb-11c5-4707-a016-e705525b22ea.mp4).